inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -356,6 +358,8 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
             });
 
     // List of enabled/disabled transformations
+    pass_config->disable<ov::pass::DisableDecompressionConvertConstantFolding>();
+    pass_config->disable<ov::pass::ConvertCompressedOnlyToLegacy>();
```

Comment:
Why do we have two transformations `DisableDecompressionConvertConstantFolding` and 
                                                               `EnableDecompressionConvertConstantFolding`? And why don't we implement just one. Now the passes for different plugins create confusion because on the hand you use `DisableDecompressionConvertConstantFolding` and on the other hand `ConvertCompressedOnlyToLegacy` registers `EnableDecompressionConvertConstantFolding`. It really over-complicates understanding the logic

Reply:
Why do we need to have two separate passes `ConvertCompressedOnlyToLegacy` and `DisableDecompressionConvertConstantFolding`? Why is not sufficient to manage just `ConvertCompressedOnlyToLegacy` overall?

Reply:
These two passes cant be merged. `DisableDecompressionConvertConstantFolding` only disables constant folding for fp16 compression `Converts` otherwise first `ConstantFolding` pass will remove all `Converts` and upgrade all constants from fp16 to fp32 and it runs as early as possible. `ConvertCompressedOnlyToLegac` on the other hand runs after `ConstantFolding` because it expects that all `ShapeOf` expressions were folded and it basically removes all `Converts` to fp16 and converts model to pure fp16 which is aligned with fp16 in v10 IR